Source/Mosa.Demo.MyWorld.x86/app.config

```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Restore the MyWorld.

Reply:
Why? There would be already HelloWorld so it would be useless to have MyWorld.

Reply:
MyWorld is a template and referenced in the documentation.